### DISCUSSION TITLE
[Question] Why are releases in FDroid and Google Play so old?

### DISCUSSION THREAD

**Discussion by avanc** [zeapo/Android-Password-Store#2807] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

While setting up a new phone, I wonder why the release in FDroid is from 2021. First fear was the the project is dead. But looking at the commit proofed me wrong :-)

### Describe the solution you'd like

Publish new releases.

### Describe alternatives you've considered

Install the old (**but flawless working**) version.

**Comment by msfjarvis** [comment, marked as answer]
External factors required major changes in the app that haven't been fully smoothed out to match the quality of the existing 1.13.x release train. The project board for the eventual 2.x release is [here](https://github.com/orgs/android-password-store/projects/1).

I no longer have the time for APS that I used to, and as a result things move significantly slower given I'm the only one doing anything on the code front now. I'd like to get APS v2 to a state where I'm comfortable announcing it as an upgrade to 1.13.x, but it's simply not there and I have no idea when I'll be able to make it happen.

**Reply by stefanv** [reply]
Still, a big thank you for your continued work on the project. I've used the old version forever, and today upgraded to the new version—for me, it works beautifully.

**Reply by 6A31** [reply]
For clarity:

Which is "the new version"? 

All I'm finding is a 3-year-old release, and the Play Store version being outdated for my android version. Am I missing something?

**Reply by stefanv** [reply]
The snapshot from https://github.com/android-password-store/Android-Password-Store/releases
